test/util.test.ts

```diff
@@ -161,7 +161,8 @@ test('test getEvalValue', () => {
 });
 
 test('test policyStringToArray', () => {
-  expect(util.policyStringToArray('p,alice,data1,read')).toEqual([['p', 'alice', 'data1', 'read']]);
+  expect(util.policyStringToArray('p,alice,data1,read#testtest')).toEqual([['p', 'alice', 'data1', 'read']]);
```

Comment:
should be ['p', 'alice', 'data1', 'read#testtest'].

Reply:
You mean we should not support inline comment?